### PR TITLE
adding low-http-server

### PR DESCRIPTION
**Pull request by jkyberneees**
Adding [low-http-server](https://github.com/jkyberneees/low-http-server) HTTP server implementation. Here we are using `0http` as routing layer.
> This implementation is also based on uWebSockets!

### CODE REVIEW

**Comment by waghanza** [comment]
t LGTM @jkyberneees.

I'll merge this implementation with the `0http` in the future, btw

**Comment by jkyberneees** [comment]
Thanks @waghanza, agreed!

File: javascript/low-http-server/config.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+framework:
+  github: jkyberneees/low-http-server
+  version: 2.1
```
Comment (waghanza) [inline javascript/low-http-server/config.yaml:3]: you need to add

```yaml
build_deps:
  - git

bin_deps:
  - gcompat # required by uWebSocket
```
@jkyberneees 

to ensure compatibility with `alpine` **containers**

File: reviewers.yaml [hunk 2]
```diff
@@ -1,6 +1,8 @@
 javascript:
   0http:
     - jkyberneees
+  low-http-server:
```
Comment (waghanza) [inline reviewers.yaml:4]: 👍

**Review by waghanza — CHANGES_REQUESTED** [review]

**Review by waghanza — APPROVED** [review]

**Comment by waghanza** [comment]
Results are up-to-date @jkyberneees 

https://github.com/the-benchmarker/web-frameworks#results

**Comment by jkyberneees** [comment]
Many thanks @waghanza, you rock!!!

**Pull request merged**